.github/workflows/build.yaml

```
@@ -0,0 +1,34 @@
+name: Kubernetes-sigs/ingress2gateway build
```

I don't think you really need the repo name in this
```suggestion
name: build
```